Automattic/woocommerce-payments
# Remove WooCommerce Payments from taking over the WooCommerce Core Settings

Fixes #4073 

#### Changes proposed in this Pull Request
Remove WooCommerce Payments from taking over the WooCommerce Core Settings  Settings and updated tests. Now when you click `WooCommerce > Settings > Payments` you will goto default WC settings page instead of WC Payments setting page.








#### Testing instructions




- Checkout to this branch
- Goto `WooCommerce > Settings > Payments`
- The default should be WC Core payment settings page
- Goto `Payments > Settings`
- This should open the WC Payments setting tab as selected on WC core settings page

-------------------


**Post merge**
